app/Services/ApiRoute.php

```diff
@@ -115,7 +114,8 @@ public function addRoute($uri, $controllerMethod, $httpMethod = 'get', $name = n
                     $controllerMethod,
                     $httpMethod
                 ) {
-                    Route::$httpMethod($uri, "$controller@$controllerMethod")->name("$prefix.$name");
+                    Route::$httpMethod($uri, "$controller@$controllerMethod")
+                        ->name("$prefix." . snake_case($controllerMethod));
```

Comment:
I made this change to fix an inconsistency in the generated name vs. the pattern that we've used historically. I can't recall the precise details, but this didn't break any tests and now the names are as I expect.

Reply:
By changing this is `$name` now unused variable ? `line 96` here should probably be what you need to change to apply the `snake_case()` method. Correct me if I am wrong here ?

Reply:
You're absolutely correct; my mistake. I think what we actually want is simply to remove the former line 96 (as in this new commit), but at the same time change the new line 96 to this:

```php
$name = is_null($name) ? snake_case($controllerMethod) : $name;
```

That gives me the results I expect, all the tests pass, and the `$name` is no longer ignored.